packages/oui-clipboard/src/clipboard.html

```diff
@@ -1,10 +1,13 @@
-<div oui-tooltip="{{$ctrl.tooltipText}}" class="oui-input-group_clipboard-container">
+<div class="oui-input-group_clipboard-container"
+    oui-tooltip="{{$ctrl.tooltipText}}"
+    ng-click="$ctrl.onClick()">
     <input class="oui-input"
         type="text"
         ng-attr-id="{{::$ctrl.id}}"
         ng-attr-name="{{::$ctrl.name}}"
-        ng-click="$ctrl.onClick()"
         ng-model="::$ctrl.model"
         contenteditable>
-    <button class="oui-button"><i class="oui-icon oui-icon-copy-normal" aria-hidden="true"></i></button>
+    <button class="oui-button">
+        <span class="oui-icon oui-icon-copy-normal"></span>
```

Comment:
`aria-hidden="true"`

Reply:
We should have a description on this button, like "Copy" or something like this :)

Reply:
Yep I already told him face to face

Reply:
I've added an `aria-label` with the tooltip text "Copy to clipboard"